src/invidious/frontend/pagination.cr

```diff
@@ -72,12 +90,18 @@ module Invidious::Frontend::Pagination
     end
   end
 
-  def nav_ctoken(locale : String?, *, base_url : String | URI, ctoken : String?)
+  def nav_ctoken(locale : String?, *, base_url : String | URI, ctoken : String?, first_page : String?)
```

Comment:
Just a slight nitpick, `first_page` should be just a bool instead.

Reply:
Yes, this was the first test that worked, so I just kept the code untouched. I will replace it in the function parameters.